m2cgen/cli.py

```diff
@@ -48,6 +49,10 @@
     choices=LANGUAGE_TO_EXPORTER.keys(),
     help="Target language",
     required=True)
+parser.add_argument(
+    "--function_name", "-fn", dest="function_name", type=str,
+    default="score",
+    help="Name of the function in the generated code (defaults to 'score')")
```

Comment:
Seems that we haven't specified default values in docstrings for other arguments. Not sure that we should do it for this one.

Reply:
I see, it is just something I would expect.

If you think it does not make sense to have it here, I am happy to get rid of it.

Reply:
Thank you!